examples/tests/igraph_closeness.c

```diff
@@ -0,0 +1,287 @@
+#include <igraph.h>
+#include <stdio.h>
+#include <math.h>
+
+/* helper function, useful for viewing output */
+void igraph_vector_t_print(igraph_vector_t *exp, igraph_vector_t *actual, int size) {
+    for (int i = 0; i < size; i++) {
+        double res = VECTOR(*exp)[i];
+        double res_2 = VECTOR(*exp)[i];
+        printf("%lf %lf\n", res, res_2);
+    }
+    printf("\n");
+}
```

Comment:
Please do not include the expected results in the code.

The expected results should to in the `.out` file. The program prints the actual results. Leave it to the test framework to compare them, and do not print superfluous information.

You can use the printing function from `test_utilities.inc` to simplify the test program and reduce code duplication

Reply:
(Note that there was also a bug in this: you print `exp` twice instead of printing `exp` and `actual`. But let's just get rid of this function and go with the method in my comment above.)

Reply:
Yet another issue here was that 
```
for (int i=0; i < size; i++)
```
is not valid C89. The variable declaration cannot be inside of the `for`. Declarations must always go to the beginning of the block.